incubator/jaeger/templates/_helpers.tpl

```diff
@@ -129,6 +129,10 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
+{{- define "jaeger.collector.sampling-filename" -}}
+{{- default "sampling.json" .Values.collector.samplingStrategiesFilename -}}
```

Comment:
isn't this a bit convoluted? it could just be the default in values.yaml (and would then match the docs) - rather than being buried into the template function?

Reply:
That's the default filename the collector is expecting. I thought it would just be easier to assign to the default and then also give the ability to overwrite the filename if someone happens to have multiple sampling files.